sdk/maps/Azure.Maps.Weather/src/Models/GeoJsonFeature.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using Azure.Core;
+
+namespace Azure.Maps.Weather.Models
+{
+    /// <summary> A valid `GeoJSON Feature` object type. Please refer to [RFC 7946](https://tools.ietf.org/html/rfc7946#section-3.2) for details. </summary>
+    [CodeGenModel("GeoJsonFeature")]
```

Comment:
We don't need `CodeGenModel` if we just want to make it internal. `CodeGenModel` is for renaming the class.